{
  "type": "issue",
  "issue": {
    "id": 302942119,
    "node_id": "MDU6SXNzdWUzMDI5NDIxMTk=",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/12628",
    "number": 12628,
    "state": "closed",
    "state_reason": "completed",
    "title": "LevelDB read failure: Corruption: block checksum mismatch always happens at 250Mb cache threshold",
    "body": "Dear friends,\r\n\r\nI am experiencing a quite weird issue when performing the initial sync with Bitcoin Core wallet 0.16.0 on my iMac.\r\n\r\nThe wallet always crashes at a specific point, exactly 250 Mb cache.\r\n\r\nI have tried to increase the cache setting to the double of what is the default value, to 900 Mb to no avail.\r\n\r\ndebug.log shows that it ALWAYS gets stuck at exactly the same point, with the wallet showing the \"Error reading from the database, shutting down\" pop up.\r\n\r\nLaunching the wallet again syncs for a little while more until it gets stuck again at exactly the same point again.\r\n\r\nDebug.log shows the following as the cause when crashing:\r\n\r\n> 2018-03-07 02:06:20 UpdateTip: new best=000000000000000000f77a9ac3b079f9df4fbc1c01382e5da665d3f86123a7fa height=485013 version=0x20000002 log2_work=87.097193 tx=254082032 date='2017-09-13 11:40:48' progress=0.826117 cache=250.3MiB(1841606txo)\r\n2018-03-07 02:06:20 LevelDB read failure: Corruption: block checksum mismatch\r\n2018-03-07 02:06:20 Corruption: block checksum mismatch\r\n2018-03-07 02:06:26 Error reading from database: Database corrupted\r\n\r\n> 2018-03-07 02:08:39 UpdateTip: new best=000000000000000000f77a9ac3b079f9df4fbc1c01382e5da665d3f86123a7fa height=485013 version=0x20000002 log2_work=87.097193 tx=254082032 date='2017-09-13 11:40:48' progress=0.826116 cache=250.3MiB(1841606txo)\r\n2018-03-07 02:08:39 LevelDB read failure: Corruption: block checksum mismatch\r\n2018-03-07 02:08:39 Corruption: block checksum mismatch\r\n2018-03-07 02:08:56 Error reading from database: Database corrupted\r\n\r\n> 2018-03-07 02:28:11 UpdateTip: new best=000000000000000000f77a9ac3b079f9df4fbc1c01382e5da665d3f86123a7fa height=485013 version=0x20000002 log2_work=87.097193 tx=254082032 date='2017-09-13 11:40:48' progress=0.826105 cache=250.3MiB(1841606txo)\r\n2018-03-07 02:28:11 LevelDB read failure: Corruption: block checksum mismatch\r\n2018-03-07 02:28:11 Corruption: block checksum mismatch\r\n\r\n\r\nThese are the specs of my iMac: late 2012, i7 3.4 Ghz processor (quad core 8 threads) 32Gb RAM.\r\n\r\nThank you so much for your help, much appreciated.",
    "user": {
      "login": "fabioganga",
      "id": 7671309,
      "node_id": "MDQ6VXNlcjc2NzEzMDk=",
      "avatar_url": "https://avatars.githubusercontent.com/u/7671309?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fabioganga",
      "html_url": "https://github.com/fabioganga",
      "followers_url": "https://api.github.com/users/fabioganga/followers",
      "following_url": "https://api.github.com/users/fabioganga/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/fabioganga/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/fabioganga/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/fabioganga/subscriptions",
      "organizations_url": "https://api.github.com/users/fabioganga/orgs",
      "repos_url": "https://api.github.com/users/fabioganga/repos",
      "events_url": "https://api.github.com/users/fabioganga/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/fabioganga/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [
      {
        "id": 234879,
        "node_id": "MDU6TGFiZWwyMzQ4Nzk=",
        "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/macOS",
        "name": "macOS",
        "description": "",
        "color": "660000",
        "default": false
      },
      {
        "id": 323697996,
        "node_id": "MDU6TGFiZWwzMjM2OTc5OTY=",
        "url": "https://api.github.com/repos/bitcoin/bitcoin/labels/Data%20corruption",
        "name": "Data corruption",
        "color": "f7c6c7",
        "default": false
      }
    ],
    "assignees": [],
    "author_association": "NONE",
    "locked": true,
    "active_lock_reason": "resolved",
    "comments": 4,
    "closed_at": "2018-10-20T07:42:17Z",
    "created_at": "2018-03-07T02:28:52Z",
    "updated_at": "2021-12-16T14:10:27Z"
  },
  "events": [
    {
      "event": "labeled",
      "id": 1507967154,
      "node_id": "MDEyOkxhYmVsZWRFdmVudDE1MDc5NjcxNTQ=",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/1507967154",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-07T02:47:45Z",
      "label": {
        "name": "Data corruption",
        "color": "f7c6c7"
      }
    },
    {
      "event": "labeled",
      "id": 1509126894,
      "node_id": "MDEyOkxhYmVsZWRFdmVudDE1MDkxMjY4OTQ=",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/1509126894",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-07T15:27:22Z",
      "label": {
        "name": "MacOSX",
        "color": "660000"
      }
    },
    {
      "event": "commented",
      "id": 371970987,
      "node_id": "MDEyOklzc3VlQ29tbWVudDM3MTk3MDk4Nw==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/371970987",
      "actor": {
        "login": "eklitzke",
        "id": 2734,
        "node_id": "MDQ6VXNlcjI3MzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2734?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/eklitzke",
        "html_url": "https://github.com/eklitzke",
        "followers_url": "https://api.github.com/users/eklitzke/followers",
        "following_url": "https://api.github.com/users/eklitzke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/eklitzke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/eklitzke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/eklitzke/subscriptions",
        "organizations_url": "https://api.github.com/users/eklitzke/orgs",
        "repos_url": "https://api.github.com/users/eklitzke/repos",
        "events_url": "https://api.github.com/users/eklitzke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/eklitzke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-09T23:07:01Z",
      "updated_at": "2018-03-10T07:05:19Z",
      "author_association": "CONTRIBUTOR",
      "body": "Can you run with `-debug=leveldb` and provide more verbose error messages? I'm also interested in seeing a copy of the corrupted table file. The name of the corrupted file should be in the logs when you use `-debug=leveldb`\r\n\r\nTo actually proceed with syncing you'll need to run with `-reindex` to rebuild your local copy of the utxo database. This will take a long time because it has to reprocess old blocks, but it won't re-download any blocks you already have locally.",
      "user": {
        "login": "eklitzke",
        "id": 2734,
        "node_id": "MDQ6VXNlcjI3MzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2734?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/eklitzke",
        "html_url": "https://github.com/eklitzke",
        "followers_url": "https://api.github.com/users/eklitzke/followers",
        "following_url": "https://api.github.com/users/eklitzke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/eklitzke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/eklitzke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/eklitzke/subscriptions",
        "organizations_url": "https://api.github.com/users/eklitzke/orgs",
        "repos_url": "https://api.github.com/users/eklitzke/repos",
        "events_url": "https://api.github.com/users/eklitzke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/eklitzke/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12628#issuecomment-371970987",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628"
    },
    {
      "event": "commented",
      "id": 375937842,
      "node_id": "MDEyOklzc3VlQ29tbWVudDM3NTkzNzg0Mg==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/375937842",
      "actor": {
        "login": "fabioganga",
        "id": 7671309,
        "node_id": "MDQ6VXNlcjc2NzEzMDk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/7671309?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fabioganga",
        "html_url": "https://github.com/fabioganga",
        "followers_url": "https://api.github.com/users/fabioganga/followers",
        "following_url": "https://api.github.com/users/fabioganga/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fabioganga/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fabioganga/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fabioganga/subscriptions",
        "organizations_url": "https://api.github.com/users/fabioganga/orgs",
        "repos_url": "https://api.github.com/users/fabioganga/repos",
        "events_url": "https://api.github.com/users/fabioganga/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fabioganga/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-25T01:18:26Z",
      "updated_at": "2018-03-25T01:24:24Z",
      "author_association": "NONE",
      "body": "Hi @eklitzke and thanks for your interest in solving my issue. Apologies for the delay, I saved the entire folder containing the files producing this error so I could get back to it at a later stage but in the meanwhile I synced from scratch importing the private keys to a new wallet that is now running fine.\r\n\r\nI have now forced the client to use this corrupt data directory because I hope that my experience will help solving the issue that could occur to others.\r\n\r\nI have therefore run the client as you suggested with -debug=leveldb and here are the error messages obtained this time:\r\n\r\n[debug.log](https://github.com/bitcoin/bitcoin/files/1844934/debug.log)\r\n\r\ndb.log that is referenced to is totally blank, no information at all.\r\n\r\nI can see that it makes a reference to .dat files, please let me know which one you would like to analyze and I will happily upload it for you.\r\n\r\nMany thanks ",
      "user": {
        "login": "fabioganga",
        "id": 7671309,
        "node_id": "MDQ6VXNlcjc2NzEzMDk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/7671309?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fabioganga",
        "html_url": "https://github.com/fabioganga",
        "followers_url": "https://api.github.com/users/fabioganga/followers",
        "following_url": "https://api.github.com/users/fabioganga/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fabioganga/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fabioganga/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fabioganga/subscriptions",
        "organizations_url": "https://api.github.com/users/fabioganga/orgs",
        "repos_url": "https://api.github.com/users/fabioganga/repos",
        "events_url": "https://api.github.com/users/fabioganga/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fabioganga/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12628#issuecomment-375937842",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628"
    },
    {
      "event": "mentioned",
      "id": 1539427303,
      "node_id": "MDE0Ok1lbnRpb25lZEV2ZW50MTUzOTQyNzMwMw==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/1539427303",
      "actor": {
        "login": "eklitzke",
        "id": 2734,
        "node_id": "MDQ6VXNlcjI3MzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2734?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/eklitzke",
        "html_url": "https://github.com/eklitzke",
        "followers_url": "https://api.github.com/users/eklitzke/followers",
        "following_url": "https://api.github.com/users/eklitzke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/eklitzke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/eklitzke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/eklitzke/subscriptions",
        "organizations_url": "https://api.github.com/users/eklitzke/orgs",
        "repos_url": "https://api.github.com/users/eklitzke/repos",
        "events_url": "https://api.github.com/users/eklitzke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/eklitzke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-25T01:18:26Z"
    },
    {
      "event": "subscribed",
      "id": 1539427304,
      "node_id": "MDE1OlN1YnNjcmliZWRFdmVudDE1Mzk0MjczMDQ=",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/1539427304",
      "actor": {
        "login": "eklitzke",
        "id": 2734,
        "node_id": "MDQ6VXNlcjI3MzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2734?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/eklitzke",
        "html_url": "https://github.com/eklitzke",
        "followers_url": "https://api.github.com/users/eklitzke/followers",
        "following_url": "https://api.github.com/users/eklitzke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/eklitzke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/eklitzke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/eklitzke/subscriptions",
        "organizations_url": "https://api.github.com/users/eklitzke/orgs",
        "repos_url": "https://api.github.com/users/eklitzke/repos",
        "events_url": "https://api.github.com/users/eklitzke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/eklitzke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-25T01:18:26Z"
    },
    {
      "event": "commented",
      "id": 376434604,
      "node_id": "MDEyOklzc3VlQ29tbWVudDM3NjQzNDYwNA==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/376434604",
      "actor": {
        "login": "eklitzke",
        "id": 2734,
        "node_id": "MDQ6VXNlcjI3MzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2734?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/eklitzke",
        "html_url": "https://github.com/eklitzke",
        "followers_url": "https://api.github.com/users/eklitzke/followers",
        "following_url": "https://api.github.com/users/eklitzke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/eklitzke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/eklitzke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/eklitzke/subscriptions",
        "organizations_url": "https://api.github.com/users/eklitzke/orgs",
        "repos_url": "https://api.github.com/users/eklitzke/repos",
        "events_url": "https://api.github.com/users/eklitzke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/eklitzke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-03-27T08:07:14Z",
      "updated_at": "2018-03-27T08:07:14Z",
      "author_association": "CONTRIBUTOR",
      "body": "I have some notes in #12690 about LeveLDB corruption, but this seems to be a separate issue. The error message \"block checksum mismatch\" implies that LevelDB handed out data with checksum X to the kernel, but then later read that data back with checksum Y. That could be bad hardware, a bad filesystem, or anywhere in-between. But I don't think it's a bug in Bitcoin (or LevelDB).",
      "user": {
        "login": "eklitzke",
        "id": 2734,
        "node_id": "MDQ6VXNlcjI3MzQ=",
        "avatar_url": "https://avatars.githubusercontent.com/u/2734?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/eklitzke",
        "html_url": "https://github.com/eklitzke",
        "followers_url": "https://api.github.com/users/eklitzke/followers",
        "following_url": "https://api.github.com/users/eklitzke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/eklitzke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/eklitzke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/eklitzke/subscriptions",
        "organizations_url": "https://api.github.com/users/eklitzke/orgs",
        "repos_url": "https://api.github.com/users/eklitzke/repos",
        "events_url": "https://api.github.com/users/eklitzke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/eklitzke/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12628#issuecomment-376434604",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628"
    },
    {
      "event": "closed",
      "id": 1916100405,
      "node_id": "MDExOkNsb3NlZEV2ZW50MTkxNjEwMDQwNQ==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/1916100405",
      "actor": {
        "login": "fanquake",
        "id": 863730,
        "node_id": "MDQ6VXNlcjg2MzczMA==",
        "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/fanquake",
        "html_url": "https://github.com/fanquake",
        "followers_url": "https://api.github.com/users/fanquake/followers",
        "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
        "organizations_url": "https://api.github.com/users/fanquake/orgs",
        "repos_url": "https://api.github.com/users/fanquake/repos",
        "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/fanquake/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2018-10-20T07:42:17Z"
    },
    {
      "event": "commented",
      "id": 536532777,
      "node_id": "MDEyOklzc3VlQ29tbWVudDUzNjUzMjc3Nw==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/536532777",
      "actor": {
        "login": "udovdh",
        "id": 5012213,
        "node_id": "MDQ6VXNlcjUwMTIyMTM=",
        "avatar_url": "https://avatars.githubusercontent.com/u/5012213?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/udovdh",
        "html_url": "https://github.com/udovdh",
        "followers_url": "https://api.github.com/users/udovdh/followers",
        "following_url": "https://api.github.com/users/udovdh/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/udovdh/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/udovdh/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/udovdh/subscriptions",
        "organizations_url": "https://api.github.com/users/udovdh/orgs",
        "repos_url": "https://api.github.com/users/udovdh/repos",
        "events_url": "https://api.github.com/users/udovdh/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/udovdh/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2019-09-30T12:10:18Z",
      "updated_at": "2019-09-30T12:10:18Z",
      "author_association": "NONE",
      "body": "If such things (levelDB issues) can happen on machines not overheating, not having faulty disks or faulty RAM, then why isn't bitcoind more resilient? Another charged particle can ruin your day...\r\nDo we need to move to ECC and get some RAM-producer's stawks? Or will there be a software workaround?",
      "user": {
        "login": "udovdh",
        "id": 5012213,
        "node_id": "MDQ6VXNlcjUwMTIyMTM=",
        "avatar_url": "https://avatars.githubusercontent.com/u/5012213?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/udovdh",
        "html_url": "https://github.com/udovdh",
        "followers_url": "https://api.github.com/users/udovdh/followers",
        "following_url": "https://api.github.com/users/udovdh/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/udovdh/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/udovdh/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/udovdh/subscriptions",
        "organizations_url": "https://api.github.com/users/udovdh/orgs",
        "repos_url": "https://api.github.com/users/udovdh/repos",
        "events_url": "https://api.github.com/users/udovdh/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/udovdh/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/12628#issuecomment-536532777",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/12628"
    },
    {
      "event": "locked",
      "id": 5778613674,
      "node_id": "LOE_lADOABII584SDoenzwAAAAFYbqWq",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/5778613674",
      "actor": {
        "login": "DrahtBot",
        "id": 39886733,
        "node_id": "MDQ6VXNlcjM5ODg2NzMz",
        "avatar_url": "https://avatars.githubusercontent.com/u/39886733?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/DrahtBot",
        "html_url": "https://github.com/DrahtBot",
        "followers_url": "https://api.github.com/users/DrahtBot/followers",
        "following_url": "https://api.github.com/users/DrahtBot/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/DrahtBot/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/DrahtBot/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/DrahtBot/subscriptions",
        "organizations_url": "https://api.github.com/users/DrahtBot/orgs",
        "repos_url": "https://api.github.com/users/DrahtBot/repos",
        "events_url": "https://api.github.com/users/DrahtBot/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/DrahtBot/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2021-12-16T14:10:27Z",
      "lock_reason": "resolved"
    }
  ]
}